sessionctx/variable/sysvar.go

```diff
@@ -1916,6 +1916,13 @@ var defaultSysVars = []*SysVar{
 		s.RangeMaxSize = TidbOptInt64(val, DefTiDBOptRangeMaxSize)
 		return nil
 	}},
+	{
+		Scope: ScopeGlobal | ScopeSession, Name: TiDBMergePartitionStatsConcurrency, Value: strconv.FormatInt(DefTiDBMergePartitionStatsConcurrency, 10), Type: TypeInt, MinValue: 1, MaxValue: 32,
```

Comment:
mergeConcurrency is not more than CPU core.

Reply:
look like [it](https://github.com/pingcap/tidb/blob/master/ddl/ddl.go#L660)

Reply:
use `MaxConfigurableConcurrency` instead